README.md

```diff
@@ -46,10 +46,10 @@ let route = sublimate { rq -> [String] in
 }
 ```
 
-> † Our find functions take optional IDs.\
```

Comment:
This backslash was deliberate, it forces a line-break

Reply:
`shd101wyy.markdown-preview-enhanced` shows no difference with or with the line break (due to the &gt;s). Looks like that doesn't match github's implementation perfectly. Probably ought to report that.